.github/workflows/Build.yaml

```diff
@@ -0,0 +1,146 @@
+
+            - name: Set up Python 3.x on main branch Part 1
+              if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+              uses: actions/setup-python@v2
+              with:
+                python-version: '3.9'
+
+            - name: Set up Python 3.x on main branch Part 2
+              if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+              run: |
+                # set up python requirements and scripts on main branch
+                python3 -m venv ve1
+                cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+                cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+            - name: Check for restricted files and user permissiom
+              id: check_authorization
+              run: |
+                # check for a restricted file and, if found, check user has permissiom
+                ve1/bin/check-user --api-url=${{ github.event.pull_request._links.self.href }} --user=${{ github.event.pull_request.user.login }}
+
+            - name: Check if version updated
+              id: check-version_updated
+              if: ${{ steps.check_version_in_PR.outputs.PR_includes_release == 'true' }}
+              run: |
+                # check if version file was changed
+                ve1/bin/release-checker --version=${{ steps.check_version_in_PR.outputs.PR_version }}
+
+            - name: Approve PR
+              id: approve_pr
+              if: ${{ steps.check-version_updated.outputs.updated == 'true'}}
+              uses: hmarr/auto-approve-action@v2
+              with:
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Merge PR
+              id: merge_pr
+              if: ${{ steps.check-version_updated.outputs.updated == 'true'}}
+              uses: pascalgn/automerge-action@v0.13.1
+              env:
+                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                MERGE_METHOD: squash
+                MERGE_LABELS: ""
+
+            - name: Check for PR merge
+              if: ${{ steps.check-version_updated.outputs.updated == 'true'}}
+              run: |
+                  ./ve1/bin/check-auto-merge --api-url=${{ github.event.pull_request._links.self.href }}
+
+            - name: Create the the release
+              id: create_release
+              if: ${{ steps.check-version_updated.outputs.updated == 'true'}}
+              uses: softprops/action-gh-release@v1
+              with:
+                tag_name: ${{ steps.check_version_in_PR.outputs.PR_version }}
+                body: ${{ steps.check_version_in_PR.outputs.PR_release_body }}
+              env:
+                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: start cron job to link image in quay
+              id: quay-cron
+              if: ${{ steps.check-version_updated.outputs.updated == 'true'}}
+              run: echo "Start cron job to be completed"
```

Comment:
I don't think currently we can trigger workflow from separate workflow (we could theoretically with PAT and send a API call to trigger `workflow_dispatch` or `repository_dispatch` but that requires a PAT with write permissions).

Reply:
```suggestion
              run: echo "Start cron job to be completed"

```

Reply:
This was just a place holder but removed it now.